:: initializing oneAPI environment ...
   BASH version = 4.2.46(2)-release
:: dev-utilities -- latest
:: tbb -- latest
:: compiler -- latest
:: dpl -- latest
:: mkl -- latest
:: ipp -- latest
:: mpi -- latest
:: ccl -- latest
:: vtune -- latest
:: dpcpp-ct -- latest
:: dal -- latest
:: intelpython -- latest
:: ippcp -- latest
:: debugger -- latest
:: inspector -- latest
:: clck -- latest
:: itac -- latest
:: oneAPI environment initialized ::
     using /nfsexports/SOFTWARE/intel/oneapi/mpi/2021.1.1/bin/mpirun
Using Python 3.11.8
Python path: /lustre/home/mmollo/.cache/pypoetry/virtualenvs/tesi-magistrale-jxuYGkd5-py3.11/bin/python
Traceback (most recent call last):
  File "/ibiscostorage/mmollo/tesi-magistrale/simulazioni/02_water/02_dimer/02_binding_energy/MACE-ICE13-1/binding_energy.py", line 9, in <module>
    atoms = read("init.xyz")
            ^^^^^^^^^^^^^^^^
  File "/lustre/home/mmollo/.cache/pypoetry/virtualenvs/tesi-magistrale-jxuYGkd5-py3.11/lib/python3.11/site-packages/ase/io/formats.py", line 794, in read
    format = format or filetype(filename, read=isinstance(filename, str))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/home/mmollo/.cache/pypoetry/virtualenvs/tesi-magistrale-jxuYGkd5-py3.11/lib/python3.11/site-packages/ase/io/formats.py", line 966, in filetype
    fd = open_with_compression(filename, 'rb')
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lustre/home/mmollo/.cache/pypoetry/virtualenvs/tesi-magistrale-jxuYGkd5-py3.11/lib/python3.11/site-packages/ase/io/formats.py", line 598, in open_with_compression
    return open(filename, mode)
           ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'init.xyz'
FINISHED!!!
